ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -85,8 +85,6 @@ echo "### Running tests"
 ssh $SSHOPTS root@$IP openshift-tests run "openshift/conformance/parallel" --dry-run \| grep 'Feature:ProjectAPI' \| openshift-tests run -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit -f -
 rv=$?
 
-echo "### Fetching results"
-ssh $SSHOPTS root@$IP tar -czf - /tmp/artifacts | tar -C ${ARTIFACT_DIR} -xzf - 
```

Comment:
This is required to fetch the e2e.log and junit xml produced from the openshift-tests

Reply:
This is done in the gather container

Reply:
Is there some reason to run scp twice?